### PR TITLE
Provisional core and extension files for 2021-07-15 ratifications.

### PR DESCRIPTION
**Pull request by tucotuco**
Functionality (validity of XML files) depends on https://github.com/gbif/rs.gbif.org/pull/71.

Vocabulary functionality for establishmentMeans, degreeOfEstablishment, and pathway depends on the creation of those and placement in vocabulary folder. I'm happy to create those if desired, but see https://github.com/gbif/rs.gbif.org/issues/21#issuecomment-900490420.

### CODE REVIEW

File: sandbox/extension/identification_history_2021-08_16.xml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+<?xml-stylesheet type='text/xsl' href='/style/human.xsl'?>
+<extension xmlns='http://rs.gbif.org/extension/'
+    xmlns:xsi='http://www.w3.org/2001/XMLSchema-instance' 
+    xmlns:dc='http://purl.org/dc/terms/'    
+    xsi:schemaLocation='http://rs.gbif.org/extension/ http://rs.gbif.org/schema/extension.xsd'
+    dc:title="Darwin Core Identification History" 
+    name="Identification" namespace="http://rs.tdwg.org/dwc/terms/" 
+    rowType="http://rs.tdwg.org/dwc/terms/Identification"
+    dc:issued='2021-07-15'
+    dc:relation='https://dwc.tdwg.org/terms/#identification'
```
Comment (MattBlissett) [inline sandbox/extension/identification_history_2021-08_16.xml:11]: `dc:subject` has disappeared here.

File: sandbox/extension/resource_relationship_2021-08_16.xml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<?xml-stylesheet type='text/xsl' href='/style/human.xsl'?>
+<extension xmlns='http://rs.gbif.org/extension/'
+    xmlns:xsi='http://www.w3.org/2001/XMLSchema-instance'
+    xmlns:dc='http://purl.org/dc/terms/'
+    xsi:schemaLocation='http://rs.gbif.org/extension/ http://rs.gbif.org/schema/extension.xsd'
+    dc:title="Darwin Core Resource Relationship" 
+    name="ResourceRelationship" namespace="http://rs.tdwg.org/dwc/terms/" 
+    rowType="http://rs.tdwg.org/dwc/terms/ResourceRelationship"
```
Comment (MattBlissett) [inline sandbox/extension/resource_relationship_2021-08_16.xml:9]: See https://github.com/tdwg/dwc/commit/1738b308f6c5584278b6670fab0b6875f48ee1d9 which changes the templates to use `'` throughout, just to make diffing easier.

**Comment by tucotuco** [comment]
@MattBlissett  I will make updates to address these issues.

File: sandbox/core/dwc_event_2021-08_16.xml [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+<?xml-stylesheet type='text/xsl' href='/style/human.xsl'?>
+<extension xmlns='http://rs.gbif.org/extension/'
+    xmlns:xsi='http://www.w3.org/2001/XMLSchema-instance' 
+    xmlns:dc='http://purl.org/dc/terms/'    
+    xsi:schemaLocation='http://rs.gbif.org/extension/ http://rs.gbif.org/schema/extension.xsd'
+    dc:title='Darwin Core Event' 
+    name='Event' namespace='http://rs.tdwg.org/dwc/terms/' 
+    rowType='http://rs.tdwg.org/dwc/terms/Event'
+    dc:issued="2021-07-15"
+    dc:relation='https://dwc.tdwg.org/terms/#event'
```
[the remark this answers is not in the document]
Comment (tucotuco) [reply sandbox/core/dwc_event_2021-08_16.xml:11]: Updated to the rs.tdwg.org endpoint.

File: sandbox/core/dwc_taxon_2021-08_16.xml [hunk 4]
```diff
@@ -0,0 +1,64 @@
+<?xml version='1.0' encoding='UTF-8'?>
```
[the remark this answers is not in the document]
Comment (tucotuco) [reply sandbox/core/dwc_taxon_2021-08_16.xml:1]: Error propagation. Amended

File: sandbox/extension/identification_history_2021-08_16.xml (hunk 1 above)
Comment (tucotuco) [reply sandbox/extension/identification_history_2021-08_16.xml:11]: Included now.

File: sandbox/extension/resource_relationship_2021-08_16.xml (hunk 2 above)
Comment (tucotuco) [reply sandbox/extension/resource_relationship_2021-08_16.xml:9]: Amended throughout.

**Comment by tucotuco** [comment]
@MattBlissett I think I have fixed everything you detected.

**Comment by tucotuco** [comment]
It is a good time to consider the proposal in issue #73  and remove https://github.com/gbif/rs.gbif.org/blob/master/sandbox/core/dwc_event_2021-08-16.xml#L16. I did not do so.

**Pull request merged**